src/tls.c

```diff
@@ -4855,10 +4832,16 @@ static int TLSX_KeyShare_GenEccKey(WOLFSSL *ssl, KeyShareEntry* kse)
  */
 static int TLSX_KeyShare_GenKey(WOLFSSL *ssl, KeyShareEntry *kse)
 {
+#ifndef NO_DH
     /* Named FFHE groups have a bit set to identify them. */
     if ((kse->group & NAMED_DH_MASK) == NAMED_DH_MASK)
         return TLSX_KeyShare_GenDhKey(ssl, kse);
```

Comment:
You mean include versions of `TLSX_KeyShare_GenDhKey` and `TLSX_KeyShare_GenEccKey`, but have them return `NOT_COMPILED_IN`? I would agree if those were public functions, but since they are static and not called anywhere else it seems fine as-is.